awesome/src/objects/tag.rs

```diff
@@ -95,9 +95,11 @@ impl UserData for TagState {
     }
 }
 
-pub fn init(lua: &Lua) -> rlua::Result<()> {
+pub fn init(lua: &Lua) -> rlua::Result<Class<TagState>> {
```

Comment:
The return values where removed in a recent commit because they were never used and caused errors. Did you find this unsatisfying, or are those changes only artifacts of commits on an old state?

Reply:
I found that unsatisfying, and I realise now that I shouldn't have added the unused result error on test. 

Plus it seemed weird to have a builder with no `build` method, even if it's technicaly not needed.